tests/pos/implicit-lower-bound.scala

```diff
@@ -9,6 +9,6 @@ class Test {
   def get1(implicit lf: List[_ <: Bar]) = {}
   def get2(implicit lf: List[_ >: Bar]) = {}
 
-  get1 // works
-  get2 // error
+  get1
+  get2
```

Comment:
I'll note that this line doesn't compile with scalac, so scalac interpretation of SLS 7.2 seems to match what I had in mind.